src/write_outputs/write_allamcyclelox.jl

```diff
@@ -0,0 +1,178 @@
+
+	for y in ALLAM_CYCLE_LOX
+		if y in COMMIT_Allam
+			retcapAllam_sco2turbine[y] = value.(EP[:vRET_AllamCycleLOX])[y, sco2turbine]* allam_dict[y,"cap_size"][sco2turbine]
+			retcapAllam_asu[y] = value.(EP[:vRET_AllamCycleLOX])[y, asu]* allam_dict[y,"cap_size"][asu]
+			retcapAllam_lox[y] = value.(EP[:vRET_AllamCycleLOX])[y, lox]* allam_dict[y,"cap_size"][lox]
+		else
+			retcapAllam_sco2turbine[y] = value.(EP[:vRET_AllamCycleLOX])[y, sco2turbine]
+			retcapAllam_asu[y] = value.(EP[:vRET_AllamCycleLOX])[y, asu]
+			retcapAllam_lox[y] = value.(EP[:vRET_AllamCycleLOX])[y, lox]
+		end
+	end
+
+
+	dfCapAllam = DataFrame(Resource = gen.resource[ALLAM_CYCLE_LOX],
+		Zone = gen.zone[ALLAM_CYCLE_LOX],
+		
+		StartCap_sCO2turbine_MW_gross = MultiStage == 1 ? value.(EP[:vEXISTINGCAP_AllamCycleLOX]) : [allam_dict[y, "existing_cap"][sco2turbine] for y in ALLAM_CYCLE_LOX],
+		StartCap_ASU_MW_gross = MultiStage == 1 ? value.(EP[:vEXISTINGCAP_AllamCycleLOX]) : [allam_dict[y, "existing_cap"][asu] for y in ALLAM_CYCLE_LOX],
+		StartCap_LOX_t = MultiStage == 1 ? value.(EP[:vEXISTINGCAP_AllamCycleLOX]) : [allam_dict[y, "existing_cap"][lox] for y in ALLAM_CYCLE_LOX],
+		
+		NewCap_sCO2turbine_MW_gross = capAllam_sco2turbine[ALLAM_CYCLE_LOX],
+		NewCap_ASU_MW_gross = capAllam_asu[ALLAM_CYCLE_LOX],
+		NewCap_LOX_t = capAllam_lox[ALLAM_CYCLE_LOX],
+
+		RetCap_sCO2turbine_MW_gross = retcapAllam_sco2turbine[ALLAM_CYCLE_LOX],
+		RetCap_ASU_MW_gross = retcapAllam_asu[ALLAM_CYCLE_LOX],
+		RetCap_LOX_t = retcapAllam_lox[ALLAM_CYCLE_LOX],
+
+		EndCap_sCO2turbine_MW_gross = [value.(EP[:eTotalCap_AllamcycleLOX])[y,sco2turbine] for y in ALLAM_CYCLE_LOX],
+		EndCap_ASU_MW_gross = [value.(EP[:eTotalCap_AllamcycleLOX])[y,asu] for y in ALLAM_CYCLE_LOX],
+		EndCap_LOX_t = [value.(EP[:eTotalCap_AllamcycleLOX])[y,lox] for y in ALLAM_CYCLE_LOX]
+	)
+
+
+	if setup["ParameterScale"] ==1
+		dfCapAllam.StartCap_sCO2turbine_MW_gross = dfCapAllam.StartCap_sCO2turbine_MW_gross* ModelScalingFactor
+		dfCapAllam.RetCap_sCO2turbine_MW_gross = dfCapAllam.RetCap_sCO2turbine_MW_gross * ModelScalingFactor
+		dfCapAllam.NewCap_sCO2turbine_MW_gross = dfCapAllam.NewCap_sCO2turbine_MW_gross * ModelScalingFactor
+		dfCapAllam.EndCap_sCO2turbine_MW_gross = dfCapAllam.EndCap_sCO2turbine_MW_gross * ModelScalingFactor
+
+		dfCapAllam.StartCap_ASU_MW_gross = dfCapAllam.StartCap_ASU_MW_gross* ModelScalingFactor
+		dfCapAllam.RetCap_ASU_MW_gross = dfCapAllam.RetCap_ASU_MW_gross * ModelScalingFactor
+		dfCapAllam.NewCap_ASU_MW_gross = dfCapAllam.NewCap_ASU_MW_gross * ModelScalingFactor
+		dfCapAllam.EndCap_ASU_MW_gross = dfCapAllam.EndCap_ASU_MW_gross * ModelScalingFactor
+
+		dfCapAllam.StartCap_LOX_t = dfCapAllam.StartCap_LOX_t * ModelScalingFactor
+		dfCapAllam.RetCap_LOX_t = dfCapAllam.RetCap_LOX_t * ModelScalingFactor
+		dfCapAllam.NewCap_LOX_t = dfCapAllam.NewCap_LOX_t * ModelScalingFactor
+		dfCapAllam.EndCap_LOX_t = dfCapAllam.EndCap_LOX_t * ModelScalingFactor
+	end
```

Comment:
Consider using the `scale_columns!` function like you did in `scale_allamcycle_data!`.